rockcraft.yaml

```diff
@@ -8,38 +8,36 @@ base: ubuntu:22.04
 platforms:
   amd64:
 
+run_user: _daemon_
+
+environment:
+  SPARK_HOME: /opt/spark
```

Comment:
I thought we'd use `/var/lib/spark`?

Reply:
No, this is different. I know it is called spark home but this is effectively where all the spark binaries are. The SPARK_HOME name is reserved, but it does not refer to a HOME folder